state_machine/applications/system-check/tests/nvm_access_test.py

```diff
@@ -0,0 +1,74 @@
+from lib.pycubed import cubesat
+
+nvm_counters = {"c_boot": cubesat.c_boot,
+                "c_state_err": cubesat.c_state_err,
+                "c_vbus_rst": cubesat.c_vbus_rst,
+                "c_deploy": cubesat.c_deploy,
+                "c_downlink": cubesat.c_downlink,
+                "c_logfail": cubesat.c_logfail}
+nvm_counters_items = sorted(nvm_counters.items())
+nvm_counters_values = sorted(nvm_counters.values())
+
+def verify_bits(counterstr, counter):
```

Comment:
This function feels pretty hardcoded, and will break if we change the sizes of our NVM counters.
```python
    def __init__(self,num_bits,register,lowest_bit):
        self.maxval = (1 << num_bits)-1
        self.bit_mask = self.maxval << lowest_bit
        self.lowest_bit=lowest_bit
        self.byte=register
```
Looking at the source code of multiBitFlag we should be able to extract the max value automatically.